lib/plugin.js

```diff
@@ -71,133 +77,124 @@ module.exports = {
               prevent = true
               node.remove()
           }
+
           return prevent
         },
-        end: function (node, metadata, context) {
+        end (node, metadata, context) {
           return true
         }
       },
       options: {
         expect: { self: true },
         stack: [],
-        begin: function (node, metadata, context) {
+        begin (node, metadata, context) {
           this.stack.push(util.extend({}, context.config))
           let options
           try {
             options = JSON.parse(metadata.param)
           } catch (e) {
-            throw node.error('Invlaid options object', { details: e })
+            throw node.error('Invalid options object', { details: e })
           }
+
           context.config = config.configure(options, context.config.plugins)
           context.util = util.configure(context.config)
           return true
         },
-        end: function (node, metadata, context) {
+        end (node, metadata, context) {
           const config = this.stack.pop()
           if (config && !metadata.begin) {
             context.config = config
             context.util = util.configure(context.config)
           }
+
           return true
         }
       },
       config: {
         expect: { self: true },
         expr: {
-          fn: /function([^(]*)\(([^()]*?)\)[^{]*\{([^]*)\}/ig,
+          fn: /function([^(]*)\(([^()]*?)\)[^{]*{([^]*)}/ig,
           rx: /\/([^/]*)\/(.*)/ig
         },
         stack: [],
-        begin: function (node, metadata, context) {
+        begin (node, metadata, context) {
           this.stack.push(util.extend({}, context.config))
           let configuration
           try {
-            configuration = eval('(' + metadata.param + ')') // eslint-disable-line no-eval
+            configuration = eval(`(${metadata.param})`) // eslint-disable-line no-eval
```

Comment:
This should be changed somehow, but I'm unsure how to get rid of `eval`.

Reply:
Why ?

Reply:
This might be a valid use case, but I generally avoid using eval because it's dangerous as you may also know.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval

Reply:
Yes, it can be disabled as mentioned in the [docs](https://rtlcss.com/learn/usage-guide/control-directives/#Remarks).

Reply:
Yeah, I know, I was just thinking out loud in case the same could be achieved another way, not because the linter complained 😛